allensdk/brain_observatory/behavior/data_objects/stimuli/presentations.py

```diff
@@ -284,6 +292,12 @@ def from_stimulus_file(
             fill_omitted_values=fill_omitted_values,
             coerce_bool_to_boolean=True
         )
+        if project_code is not None:
```

Comment:
Any reason why this only needs to be run in `from_stimulus_file`? What about `from_nwb` and `from_path`? They don't need the updated stimulus_block names?

Reply:
Once the data is created and then packaged into NWB, the new column will just "be there" so there's no need to add it to `from_nwb`.